extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -324,6 +400,18 @@ export class AmpAutocomplete extends AMP.BaseElement {
     );
   }
 
+  /**
+   * Takes the given input and returns the constructed server endpoint for it.
+   * For use when publishers provide "src-base" and "src-query-param" attributes
+   * in the AMP4Email context only.
+   * @param {string=} opt_input
+   * @return {string}
+   * @private
+   */
+  generateSrc_(opt_input = '') {
+    return this.srcBase_ + '?' + this.query_ + '=' + opt_input;
```

Comment:
You need to `encodeURIComponent` these.

Either use `addParamsToUrl` or do it yourself. And also rename `input` to `query`, since it's more specific.

```js
const encodedQueryKey = encodeURIComponent(this.queryKey_);
const encodedQuery = encodeURIComponent(query);
return `${this.srcBase_}?${encodedQueryKey}=${encodedQuery}`;
```